pipelines/wdl/arrays/imputation_beagle/ImputationBeagle.wdl

```diff
@@ -128,20 +149,20 @@ workflow ImputationBeagle {
 
     Array[File] chunkedVcfsWithOverlapsForImputation = GenerateChunk.output_vcf
 
-    call tasks.StoreChunksInfo as StoreContigLevelChunksInfo {
+    call beagleTasks.CountVariantsInChromosome {
```

Comment:
could we combine this task, ExtractUniqueVariantsFilteredChunk, and CountVariantsInChunks into the same task?  It feels liek these should all be very quick analyses but maybe its not?